addons/point_of_sale/static/src/js/models.js

```diff
@@ -238,14 +342,22 @@ class PosGlobalState extends PosModel {
             }
         }
     }
-    get_cashier(){
-        return this.cashier;
+
+    /**
+     * Return the current cashier (in this case, the user)
+     * @returns {name: string, id: int, role: string}
+     */
+    get_cashier() {
+        return this.user;
     }
-    set_cashier(employee){
-        this.cashier = employee;
+    get_cashier_user_id() {
+        return this.user.id;
     }
+    // set_cashier(employee){
+    //     this.cashier = employee;
+    // }
```

Comment:
I believe this commented code can be removed.

Reply:
upsie